src/main/java/com/chnu/model/Profile.java

```diff
@@ -0,0 +1,67 @@
+package com.chnu.model;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "profiles")
+@NamedQueries({
+        @NamedQuery(name = Profile.FIND_BY_PROFILE, query = "select p from Profile p where p.userId = :user_id")
```

Comment:
Why this query is needful? You can use method from AbstractRepository instead of that.